canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -112,12 +143,17 @@ class CheckboxsetControl extends React.Component {
 	}
 }
 
+CheckboxsetControl.defaultProps = {
```

Comment:
Why do we need a default prop?  Disabled comes from control states and isn't passed in like this.

Reply:
Removed them.